pkg/proto/http/oidc/oidc.proto

```diff
@@ -21,6 +21,12 @@ message CookieValue {
     // the authorization endpoint URL took place. The user will be
     // redirected back to this page upon success.
     string original_request_uri = 2;
+
+    // The value that was used to compute the code challenge that was
+    // provided to the authorization endpoint URL. It is used to implement
+    // PKCE (RFC 7636) and is sent to the token endpoint URL when
+    // exchanging the authorization code for an access token.
+    string code_verifier = 3;
```

Comment:
I think that generating 32 bytes and calling `RawURLEncoding.EncodeToString()` against it is a good way of generating a code verifier. However, as part of the cookie can we keep this a `bytes` field that holds the original non-base64 data? I know the savings aren't that big (43 vs. 32 bytes), but at least it's consistent with what we do for `state_verifier`.